src/components/LoanCollections/HomeExp/KivaLoanCardCategory.vue

```diff
@@ -213,4 +213,10 @@ export default {
 	section#loan-category-carousel :first-child {
 		column-gap: 0;
 	}
+
+	@media (max-width: 980px) {
```

Comment:
What's the significance of 980 in this rule. Our `lg:` breakpoint starts at 1024...